packages/next/src/server/app-render/create-component-tree.tsx

```diff
@@ -54,6 +54,8 @@ function errorMissingDefaultExport(pagePath: string, convention: string) {
   )
 }
 
+const cacheNodeKey = 'cache-node'
```

Comment:
Using the same key to keep the original behavior. If we want to remount the children of the fragment even if they'd reconcile against each other, we'd have to use a different key. But that wouldn't match the original behavior where the key was missing.

Reply:
@ztanner do the key changes make sense?

Reply:
If this behavior is more or less the equivalent of it being undefined previously, then yes I think so.

Reply:
> If this behavior is more or less the equivalent of it being undefined previously

"should be" ™

Reply:
let's use single letter keys for static keys representing the slot they are in. keeps the byte size down

Reply:
Yeah, we were discussing this on Slack and I thought we could do 
```
"cache-node"  -> "c"
"flight-data-path-head" -> "h"
```
zack agreed and you already proposed "h" as well so I'm going forward with that.

Reply:
Using single letter keys now:
```
"cache-node"  -> "c"
"flight-data-path-head" -> "h"
```